Dockerfile

```diff
@@ -1,19 +1,15 @@
 FROM node:lts-alpine
 
+ARG repository
 ARG name
 ARG value
 ARG token
-ARG repo
-
-ENV REPO_ACCESS_TOKEN=$token
-ENV REPO=$repo
-ENV SECRET_NAME=$name
-ENV SECRET_VALUE=$value
+ARG org
 
 COPY src ./
 
 COPY package*.json ./
 
 RUN npm install --only=production
 
-CMD ["node", "secret.js"]
+CMD ["node", "secret.js", "$repository", "$name", "$value", "$token", "$org"]
```

Comment:
I don't undestand why you are using docker to run a nodejs action. Github suports running nodejs without the docker overhead.

https://docs.github.com/en/actions/language-and-framework-guides/using-nodejs-with-github-actions

https://github.com/imobanco/actions-set-secret/blob/077e9328dd468bb73526b3bb9240deb18b0d988f/action.yml#L40